src/main/java/Ui.java

```diff
@@ -0,0 +1,251 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+/**
+ * Ui class interacts with Duke user, requesting input and printing prompts for user.
+ *
+ * This class provides constructor for Ui object.
+ */
+public class Ui {
+    private final String BORDER = "**********************************************************************";
+
+    /**
+     * Constructor for Ui object.
+     */
+    public Ui() {
+    }
+
+    /**
+     * Ui object displays welcome message when Duke is launched.
+     */
+    protected void showWelcome() {
+        String greeting = "Hi! My name is Alfred. I aim to please.";
+        System.out.println(BORDER + "\n  " + greeting + "\n" + BORDER);
+    }
+
+    /**
+     * Ui object displays error message if any error arises when loading from save file
+     * from hard disk.
+     */
+    protected void showLoadingError() {
+        System.out.println();
+    }
+
+    /**
+     * Ui object displays input String.
+     * @param toPrint input String for Ui object to display.
+     */
+    protected void print(String toPrint) {
+        System.out.println(BORDER + toPrint + BORDER);
+    }
+
+    /**
+     * Begin Ui's interaction with user.
+     */
+    protected void start(){
+        // Create new Scanner object to read user input.
+        Scanner sc = new Scanner(System.in);
+
+        // Initialise variables needed
+        String input = sc.next();
+        String taskAdd = "Noted. The following task has been added:\n   ";
+        String[] split;
+        String description;
+
+        // Check if user is done with Duke
+        while (!input.equals("bye")) {
```

Comment:
The command 'bye' should be treated equally as other command to avoid confusion in future development